libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/lifecycle/NavigationScaleGestureHandlerOptions.kt

```diff
@@ -56,10 +56,12 @@ class NavigationScaleGestureHandlerOptions private constructor(
 
         if (context != other.context) return false
         if (followingInitialMoveThreshold != other.followingInitialMoveThreshold) return false
-        if (followingMultiFingerMoveThreshold != other.followingMultiFingerMoveThreshold)
+        if (followingMultiFingerMoveThreshold != other.followingMultiFingerMoveThreshold) {
```

Comment:
I'd find it annoying if I had to manually adjust codestyle of a generated method after every regenerations. WDYT? Can these `equals` methods be excluded from checks somehow?

Reply:
actually you can run gradle task `kotlinFormat`, that do everything for you

Reply:
Still, generated code should generally pass the linter, otherwise generation should be adjusted. Since in this case we rely on built-in tooling, is there a way to skip the check? Or would we have to dive deeper to provide custom generation templates?

Reply:
actually, for me, it always works like this. On my local machine auto-generated code has a bit different style than ktlint expected (for instance no spaces around logical operator were run via auto-generator). So I have to run `ktlintFormat` anytime. 

Let's try to address this issue complex and sync code styles between ktlint, IDE, and all developers, I cut `#NAVAND-685`